snap/snapcraft.yaml

```diff
@@ -145,23 +145,51 @@ parts:
       - libwayland-cursor0
       - libwayland-egl1
       - libwayland-server0
-      - libnvidia-egl-wayland1
     prime:
       - usr/lib
       - usr/share/doc/*/copyright
-      - usr/share/egl/egl_external_platform.d
 
-  gbm:
-    # GBM EGL external platform
-    plugin: nil
+  nvidia-egl-ext-deps:
+    # eglexternalplatform-dev >= 1.2
```

Comment:
I think this comment should go, as it's actually `== 1.2` below, or you'd have to explain more that this _has_ to be `>= 1.2`.

Reply:
Removed